src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class main  {
```

Comment:
The class name `main` should be capitalized to `Main` to follow Java naming conventions.